tests/services/phase.service.spec.ts

```diff
@@ -19,17 +19,17 @@ describe('PhaseService', () => {
   });
 
   describe('.storeSessionData()', () => {
-    it('should return an Observable of true if an openPhase is defined', () => {
+    it('should return an Observable of void if an openPhase is defined', () => {
       githubService.fetchSettingsFile.and.returnValue(of(BUG_REPORTING_PHASE_SESSION_DATA));
-      phaseService.storeSessionData().subscribe((result: boolean) => {
```

Comment:
Instead of checking whether it returns an observable of `void`, we can consider checking if no error is thrown in this pipeline. That way, these test cases would be most useful as a result! 

What do you think? @zatkiller @CATcher-org/2021-devs

Reply:
True, I think that would make more sense too. I will make the necessary changes